bundles/io/org.openhab.io.jetty.certificate/src/main/java/org/openhab/io/jetty/certificate/internal/CertificateGenerator.java

```diff
@@ -0,0 +1,198 @@
+package org.openhab.io.jetty.certificate.internal;
```

Comment:
This one ? 
Bundle-License: http://www.eclipse.org/legal/epl-v10.html;description=Eclipse Public License Version 1.0;link=http://www.eclipse.org/legal/eplfaq.php